CHANGELOG.next.asciidoc

```diff
@@ -257,6 +257,8 @@ automatic splitting at root level, if root level element is an array. {pull}3415
 
 *Elastic Logging Plugin*
 
+*Libbeat*
+- Fix ndjson parser to store JSON fields correctly under `target` {issue}29395[29395]
```

Comment:
We don't have a section for libbeat in the CHANGELOG.  Instead we have a section titled "Affecting all Beats".  This entry should go there.  Specifically, this is the line where the entry should go: https://github.com/elastic/beats/blob/4eaa9180deac7c44be3c45f9e1f35d405cbaf439/CHANGELOG.next.asciidoc?plain=1#L77